virttest/qemu_vm.py

```diff
@@ -3101,6 +3101,9 @@ def create(self, name=None, params=None, root_dir=None,
                                                 "[9p proxy helper]",
                                                 auto_close=False)
             else:
+                for dev in self.devices:
+                    if isinstance(dev, qdevices.QDaemonDev):
+                        dev.start_daemon()
```

Comment:
Suggest wrapping these lines as a new method of `DevContainer`.

Reply:
Updated, thanks.